typescript/src/live/helpers.ts

```diff
@@ -33,6 +33,31 @@ export function printMessage(message: {
   }
 }
 
+export function printMessagePartial(message: {
```

Comment:
Update the existing printMessage ?  
if there is no partial, it will print do a console.log live the original printMessage